package.json

```diff
@@ -40,40 +40,41 @@
   "devDependencies": {
     "@google-cloud/storage": "^1.2.1",
     "@std/esm": "^0.5.1",
-    "archiver": "^2.0.0",
-    "babylon": "^6.17.4",
-    "bytes": "^2.5.0",
-    "chai": "^4.1.0",
-    "chalk": "^2.0.1",
+    "archiver": "^2.0.3",
+    "babylon": "^6.18.0",
+    "bytes": "^3.0.0",
+    "chai": "^4.1.2",
+    "chalk": "^2.1.0",
     "clear-require": "^2.0.0",
     "common-tags": "^1.4.0",
     "coveralls": "^2.13.1",
-    "eslint": "^4.3.0",
+    "eslint": "^4.6.1",
     "eslint-config-google": "^0.9.1",
     "fs-extra": "^4.0.1",
-    "github": "^9.2.0",
+    "github": "^11.0.0",
     "glob": "^7.1.2",
     "gulp": "github:gulpjs/gulp#4.0",
     "gulp-rename": "^1.2.2",
-    "gulp-sourcemaps": "^2.6.0",
-    "gzip-size": "^3.0.0",
-    "lerna": "^2.0.0",
-    "mocha": "^3.5.0",
-    "nyc": "^11.1.0",
-    "path-extra": "^4.2.0",
+    "gulp-sourcemaps": "^2.6.1",
+    "gzip-size": "^4.0.0",
+    "lerna": "^2.1.2",
+    "mocha": "^3.5.3",
+    "nyc": "^11.2.1",
+    "path-extra": "^4.2.1",
     "pr-bot": "0.2.0",
     "pre-push": "^0.1.1",
-    "rollup": "^0.49.2",
-    "rollup-plugin-commonjs": "^8.0.2",
+    "rollup": "^0.49.3",
+    "rollup-plugin-commonjs": "^8.2.1",
     "rollup-plugin-istanbul": "^1.1.0",
     "rollup-plugin-multi-entry": "^2.0.1",
     "rollup-plugin-node-resolve": "^3.0.0",
-    "rollup-plugin-replace": "^1.1.1",
+    "rollup-plugin-replace": "^2.0.0",
     "rollup-plugin-uglify-es": "0.0.1",
-    "rollup-stream": "^1.23.1",
+    "rollup-stream": "^1.24.1",
     "semver": "^5.4.1",
     "service-worker-mock": "^1.2.2",
-    "sinon": "^2.4.1",
+    "sinon": "^3.2.1",
+    "text-encoding": "^0.6.4",
```

Comment:
I'm fine all plugins, where did this come from? Can't see it used anywhere.

Reply:
Good question. Looks like it's a [dependency](https://github.com/sinonjs/sinon/blob/master/package.json#L45) of `sinon` and my local `node_modules` must have been in a weird state, because it wasn't getting installed until I manually listed it. I'll remove it.